src/ChartInternal/shape/shape.ts

```diff
@@ -204,6 +204,8 @@ export default {
 				value = $$.getRatio("index", d, true);
 			} else if ($$.isBubbleZType(d)) {
 				value = $$.getBubbleZData(d.value, "y");
+			} else if (isRange(value)) {
```

Comment:
`isRange` is too broad, because `[n, n]` ranged array value used only in `bar` type. In this condition, when data is given with `[n, n]` format, but not being "bar" type will meet the condition.

So, rather than "isRange()", determining ranged bar type value should work only for bar type like `.isBubbleZTtype()`.

Reply:
@netil 

How about naming `.isBarRangeType(d)` ?
like https://github.com/naver/billboard.js/blob/f84ab3e9c60dd17b233992fea927e79b6e4b4530/src/ChartInternal/internals/type.ts#L158-L160